crates/model/src/order.rs

```diff
@@ -291,8 +282,7 @@ pub struct OrderData {
     /// as they should only ever be used to improve the price of a regular order
     /// and should not be settled on their own.
     /// This is 0 for limit orders as their fee gets taken from the surplus.
-    /// This is `OrderMetadata::full_fee_amount` modulo possible subsidies for
-    /// market orders.
+    /// This is equal to `OrderMetadata::full_fee_amount`
```

Comment:
I think we should rather say that for old orders these values might be different due to subsidies. Otherwise people might investigate the behavior of old orders and get surprised that this value is not actually equal.

Reply:
Makes sense yes, old orders might contain different values.